### PR TITLE
Utils Refactor

### PR DESCRIPTION
**Pull request by kevinhughes27**
While testing @millerda's branch for #3 I made the following refactors:

remove prepare_image and clean up resize_image. I stepped through a full cycle to ensure everything still works
  * preparing samples
  * training
  * playing

Next I split the Screenshot class into 2 classes adding a Sample class which represent the image after it has been transformed to be a sample.

I also removed the perfTests since they didn't actually use the utils there is no real reason to keep them around.

@bzier can you have a look as well

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -20,42 +17,28 @@
...
     OFFSET_X = 0
     OFFSET_Y = 0
 
+
+class Sample:
     IMG_W = 200
     IMG_H = 66
     IMG_D = 3
 
-    image_array = array.array('B', [0] * (SRC_W * SRC_H * SRC_D));
```
Comment (kevinhughes27) [inline utils.py]: also this doesn't seem to be used anywhere

**Comment by bzier** [comment]
Hey, I've been out of town for the last week. I'll take a look at this stuff when I get back, or this weekend.

**Comment by bzier** [comment]
Hey Kevin, sorry I never got around to reviewing this. I've been really busy the last couple months and haven't spent any time on this project. I'd like to get back to it, but don't know how soon that might happen.

**Comment by kevinhughes27** [comment]
No worries! If I broke something I'm sure someone will report it :)

**Pull request merged**